scss/standard/_bootswatch.scss

```diff
@@ -20,3 +20,8 @@
 .pagination {
   margin: .5rem;
 }
+
+.btn.btn-link {
+  padding: 0;
+  vertical-align: baseline;
```

Comment:
I'd experiment with this, but you might want to ignore the font that .btn adds by doing

font-size: inherit;

Might fix any issues with btn-links in header tags and stuff

Reply:
```
.btn.btn-link {
  font-size: inherit;
  padding: 0;
  vertical-align: baseline;

  &:focus,
  &:hover {
    border: 0;
  }
}
```

Reply:
Adding `font-size: inherit;` did allow me to simplify my changes to the slide widgets in bits so that has been added.

For the border, why do we only want it to be 0 on hover and focus and not all the time?

Reply:
I was assuming bootstrap adds it on hover and focus. so if you dont override it there it might not work? its already not there normally